Soundex.Tests.cpp

```diff
@@ -1,9 +1,44 @@
 #include <gtest/gtest.h>
 #include "Soundex.h"
 
-TEST(SoudexTestsuite, ReplacesConsonantsWithAppropriateDigits) {
- //AAA
-  char soundex[5];
-  generateSoundex("AX", soundex);
- // ASSERT_EQ(soundex,"A200");
+TEST(SoundexTestsuite, ReplacesConsonantsWithAppropriateDigits) {
+    char soundex[5];
```

Comment:
Explore Data Driven testing in Gtest to reduce Test Code

Reply:
Sure, will explore that